solr/modules/monitor/src/java/org/apache/solr/monitor/search/ReverseSearchHandler.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ *  * Licensed to the Apache Software Foundation (ASF) under one or more
+ *  * contributor license agreements.  See the NOTICE file distributed with
+ *  * this work for additional information regarding copyright ownership.
+ *  * The ASF licenses this file to You under the Apache License, Version 2.0
+ *  * (the "License"); you may not use this file except in compliance with
+ *  * the License.  You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package org.apache.solr.monitor.search;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.solr.handler.component.QueryComponent;
+import org.apache.solr.handler.component.SearchHandler;
+
+public class ReverseSearchHandler extends SearchHandler {
+
+  @Override
+  protected List<String> getDefaultComponents() {
+    ArrayList<String> names = new ArrayList<>(2);
+    names.add(QueryComponent.COMPONENT_NAME);
```

Comment:
Added https://github.com/apache/solr/pull/2382/commits/bf0daf77ec278a20f274a7a0a847b1765a4c1cd9 commit to explore _not_ configuring the query component.
```suggestion
```